src/observables/IteratorObservable.ts

```diff
@@ -173,10 +173,6 @@ function numberIsFinite(value) {
   return typeof value === 'number' && root.isFinite(value);
 }
 
-function isNan(n) {
-  return n !== n;
-}
```

Comment:
Is there any particular usecase for this function? It's currently removed since there isn't any reference but functional behavior seems bit weird, curious if there is specific intended purpose.